history-network.md

```diff
@@ -60,31 +60,37 @@ Each of these data items are represented as a key-value pair. Denote the key for
 
 All `content` items are transmitted as RLP-encoded byte arrays.
 
-We derive a `content-id` from the `content-key` as `H(content-key)` where `H` denotes the SHA-256 hash function, which outputs 32-byte values. The `content-id` represents the key in the DHT that we use for `distance` calculations.
-
-All `content-key` values are encoded and decoded according to SSZ sedes.
+All `content-key` values are encoded and decoded as an [`SSZ Union`](https://github.com/ethereum/consensus-specs/blob/dev/ssz/simple-serialize.md#union) type.
+```
+content-key = Union[blockheader, blockbody, receipt]
+serialized-content_key = serialize(content_key)
+```
 
 #### Block Header
 
 ```
+selector = 0x00
 content-key  = Container(chain-id: uint16, content-type: uint8, block-hash: Bytes32)
```

Comment:
The `content-type` should be removed from the container, as it is now basically the selector in the Union.

Reply:
Agreed, fixed.